fastlane.gemspec

```diff
@@ -31,6 +31,10 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'addressable', '~> 2.3.8' # Support for URI templates
   spec.add_dependency 'xcode-install', '~> 1.0.1' # Download new Xcode versions
 
+
+  spec.add_dependency 'erubis', '~> 2.7.0'
```

Comment:
Is there any reason we can't use erb from the Ruby standard library? https://github.com/fastlane/fastlane/blob/master/lib/fastlane/junit_generator.rb#L12